### PR TITLE
Phone number hint API update

### PR DESCRIPTION
**Pull request by feduke-nukem**
Updated phone number hint API

Possibly resolving issue: https://github.com/surfstudio/flutter-otp-autofill/issues/9

New API look: 
image

### CODE REVIEW

**Comment by feduke-nukem** [comment]
> @feduke-nukem edit change list, pls

done

**Review by dshevchenkoo — APPROVED** [review]

**Comment by dshevchenkoo** [comment]
@feduke-nukem Conflict

File: android/src/main/kotlin/ru/surfstudio/otp_autofill/OTPPlugin.kt [hunk 1]
```diff
@@ -93,21 +94,21 @@ public class OTPPlugin : FlutterPlugin, MethodCallHandler, PluginRegistry.Activi
...
-            activity!!.startIntentSenderForResult(
-                    intent.intentSender,
-                    credentialPickerRequest,
-                    null, 0, 0, 0
-            )
-        }
+        
+        if(activity == null) return
+
+        Identity.getSignInClient(activity!!)
```
[the remark this answers is not in the document]
Comment (feduke-nukem) [reply android/src/main/kotlin/ru/surfstudio/otp_autofill/OTPPlugin.kt:102]: done

> add comment pls

**Comment by feduke-nukem** [comment]
> @feduke-nukem Conflict

done

**Review by dshevchenkoo — APPROVED** [review]

**Pull request merged**